.github/workflows/ci_cron_daily.yml

```diff
@@ -4,16 +4,22 @@ on:
   schedule:
     # run every day at 3am UTC
     - cron: '0 3 * * *'
+  pull_request:
+    # We also want this workflow triggered if the 'Extra CI' label is added
+    # The first two are subset of the defaults
+    types:
+      - synchronize
+      - reopened
+      - labeled
 
 env:
   ARCH_ON_CI: "normal"
   IS_CRON: "true"
 
 jobs:
   tests:
-    name: ${{ matrix.name }}
```

Comment:
The name doesn't get evaluated if the job is skipped, leading to a very ugly skipped check in the PR list. Like GH makes the name out of all the variables in the matrix.

Reply:
Is there a way for me to see what the names look like now after you made this change?

Reply:
Adding the Extra CI label should do it...